### DISCUSSION TITLE
Issues setting up f1-dash on computer and mobile device

### DISCUSSION THREAD

**Discussion by f1bug** [slowlydev/f1-dash#184] [General]
Hi,

I am running the application on my computer and want to access it from my mobile. All the pages work well, but the livetiming feature doesn’t load. Initially, I thought it was due to the absence of data, but Las Vegas confirmed that this is not the issue.

I suspect the problem is related to the .env files. I have tried multiple configurations but still haven’t figured out the correct setup.

I have understand that there are two .env relevant files:

/.env

```
# the address of the live backend, found in crates/live
LIVE_BACKEND_ADDRESS=localhost:4000

# the origin of the frontend, found in dash/
ORIGIN=http://localhost:3000
```

/dash/.env
`NEXT_PUBLIC_LIVE_SOCKET_URL=http://localhost:4000`

When I run the application locally on my computer, the livetiming feature works perfectly. However, it doesn’t work on my mobile because I cannot access the computer's localhost from my mobile device unless I use the public IP address. When I replace localhost with my computer's public IP address, I encounter several errors, with the most common being:
`GET https://<IP>:4000/api/sse net::ERR_SSL_PROTOCOL_ERROR` 

Maybe I have to change the code in live to modify cors but you are runninng it in a server and make avalibale for everyone with the same code....

**Comment by slowlydev** [comment]
Hey there.

I think the issue is the addresses set in env.

When I setup f1-dash so I can access it on other devices on my network, I set the following envs.


`f1-dash/.env`
```
# 0.0.0.0 exposes the services on the whole network

API_BACKEND_ADDRESS=0.0.0.0:4001

LIVE_BACKEND_ADDRESS=0.0.0.0:4000

# your desktops ip, or what I prefer to do is "hostname.local", replace hostname with your computes hostname
ORIGIN=http://hostname.local:3000 
```

`f1-dash/dash/.env`
```
NEXT_PUBLIC_LIVE_SOCKET_URL=http://hostname.local:4000
NEXT_PUBLIC_API_URL=http://hostname.local:4001
```

Not sure why you are getting SSL errors, about that just make sure you don't use `https` anywhere always `http`.

**Reply by f1bug** [reply]
It works for me, thanks!
